cmd/commands/cli/command.go

```diff
@@ -376,21 +377,26 @@ func (c *cliApp) payout(argsString string) {
 	action := args[0]
 	switch action {
 	case "set":
-		payoutSignature := "payout set <identity> <ethAddress>"
+		payoutSignature := "payout set <identity> <ethAddress> <referralCode>"
 		if len(args) < 2 {
 			info("Please provide identity. You can select one by pressing tab.\n", payoutSignature)
 			return
 		}
 
 		var identity, ethAddress string
-		if len(args) == 3 {
+		if len(args) > 2 {
```

Comment:
I don't thing this field is required. I dont see why person can't get payout wo refferal

Reply:
Actually it could be different command also: `refferal set <identity> <referralCode>`

Reply:
It is optional:)

Reply:
my mistake